app/app_test.go

```diff
@@ -165,6 +165,53 @@ func TestDeploymentModelCreateDeployment(t *testing.T) {
 
 			OutputBody: true,
 		},
+		"ok with group": {
+			InputConstructor: &model.DeploymentConstructor{
+				Name:         pointers.StringToPointer("group"),
+				ArtifactName: pointers.StringToPointer("App 123"),
+			},
+
+			InvDevices: []model.InvDevice{
+				{
+					ID: "b532b01a-9313-404f-8d19-e7fcbe5cc347",
+				},
+			},
+			TotalCount: 1,
+
+			OutputBody: true,
+		},
+		"ok with group, two pagese": {
+			InputConstructor: &model.DeploymentConstructor{
+				Name:         pointers.StringToPointer("group"),
+				ArtifactName: pointers.StringToPointer("App 123"),
+			},
+
+			InvDevices: []model.InvDevice{
+				{
+					ID: "b532b01a-9313-404f-8d19-e7fcbe5cc347",
+				},
+			},
+			TotalCount: 2,
```

Comment:
how can that be possible when total count is 2 and there is one device returned?